packages/runtime/runtime-definitions/src/protocol.ts

```diff
@@ -48,10 +48,10 @@ export interface ISignalEnvelope {
 
 /**
  * Represents ISignalMessage with its type.
- * @public
+ * @alpha
  */
 export interface IInboundSignalMessage extends ISignalMessage {
-	type: string;
+	readonly type: string;
```

Comment:
If one of the other reviewers can add context here, this seems like a convenient time to add documentation to this, since we're changing the signature 🙂